slidez-fe-core/src/pages/dashboard/Dashboard.tsx

```diff
@@ -83,6 +84,25 @@ const Dashboard = () => {
         })
     }
 
+    const filteredPresentation = MOCK_DATA.filter((presentation) => {
+        return presentation.name
+            .toLowerCase()
+            .includes(searchField.toLowerCase())
+    })
```

Comment:
Please, use useEffect on searchField changes to filter presentations and store filtered presentations using useState()